README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">BPMN Visualization</h1>
 <p align="center">
-    <p align="center"> <img title="BPMN Visualization" src="docs/diagram-example.png" width="812" height="170"></p>
+    <p align="center"> <img title="BPMN Visualization" src="docs/diagram-hacktoberfest-example.png"></p>
```

Comment:
:question: can we only rename the original image to keep an history of image content?

Reply:
The image is still use by the html doc.

We decide to keep the both images. No rename.

Reply:
ok, this makes sense